beans/src/main/java/fr/pilato/elasticsearch/crawler/fs/beans/Doc.java

```diff
@@ -45,11 +46,13 @@
     private Path path;
     private Attributes attributes;
     private Map object;
+    private Map<String, Object> external;
 
     public Doc() {
         meta = new Meta();
         file = new File();
         path = new Path();
+        external = new HashMap<>();
```

Comment:
I wonder if this should be `null` by default so it won't generate an empty property `external` when not needed?